scripts/read-env-file.ts

```diff
@@ -0,0 +1,160 @@
+import { readFileSync, readFile } from 'fs'
+import { isAbsolute, resolve } from 'path'
+
+export interface EnvVars {
+  [key: string]: string | undefined
+}
+
+function mapToObject(map: Map<string, string>) {
+  const arr = [...map.entries()]
+  return arr.reduce((carrier: any, entry) => {
+    const [k, v] = entry
+    carrier[k] = v
+    return carrier
+  }, {})
+}
+
+function isValueQuoted(value: string): boolean {
+  return ["'", '"', '`'].some((quote) => {
+    return value.startsWith(quote) && value.endsWith(quote)
+  })
+}
+
+function stripSurroundingQuotes(value: string): string {
+  let result = value
+  if (isValueQuoted(value)) {
+    result = value.slice(1, value.length - 1)
+  }
+  return result
+}
+
+function normalizePath(path?: string): string {
+  path = path || resolve(process.cwd(), '.env')
+  return isAbsolute(path) ? path : resolve(process.cwd(), path)
+}
+
+function checkInvalidFormat(key: string, value: string, index: number) {
+  const undef = !key
```

Comment:
Copied from read-env-file npm package and changed here to be able get key without values